views/courses/components/topics.hbs

```diff
@@ -44,7 +45,8 @@
 								<i class="fa fa-ellipsis-v"></i>
 							</a>
 							<a class="btn-hidden-toggle" href="/courses/{{../courseId}}/topics/{{../_id}}"
-								title="{{$t "courses._course.topic.text.hideTopic"}}">
+								title="{{$t "courses._course.topic.text.hideTopic"}}"
+								role="button">
```

Comment:
why do we have the role="button" on an anchor?

Reply:
The anchors given the button role here act as a button. Screenreaders etc. know due to the role tag, that this can be handled as if it was a button:
![anchors](https://user-images.githubusercontent.com/45626086/94283292-1ea84700-ff51-11ea-9cb6-d542ec80ee83.png)
Changing those to real buttons would, as mentioned above require a lot of refactoring... :/